synapse/config/registration.py

```diff
@@ -108,6 +112,10 @@ def default_config(self, **kwargs):
             - vector.im
             - riot.im
 
+        # If enabled, user IDs, display names and avatar URLs will be replicated
+        # to this server whenever they change.
```

Comment:
we need to say that this is an experimental addition to the identity server API for supporting global user directories, and that the targets here should be identity server hostnames (otherwise folks will assume it's for HSes)

Reply:
Done, although it's in the /_matrix/federation path so I've said it's an API that's implemented by sydent rather than part of the IS API as such.